types/svg-path-contours/index.d.ts

```diff
@@ -0,0 +1,24 @@
+type MoveCommand = ["m" | "M", number, number];
+type LineCommand = ["l" | "L", number, number];
+type HorizontalCommand = ["h" | "H", number];
+type VerticalCommand = ["v" | "V", number];
+type ClosePathCommand = ["z" | "Z"];
+type BezierCurveCommand = ["c" | "C", number, number, number, number, number, number];
+type FollowingBezierCurveCommand = ["s" | "S", number, number, number, number];
+type QuadraticCurveCommand = ["q" | "Q", number, number, number, number];
```

Comment:
is there a gain for users to use named tuples? so here:
```suggestion
type QuadraticCurveCommand = ["q" | "Q", cpx: number, cpy: number, x: number, y: number];
```
for consideration

Reply:
Thanks! I did not know about named tuples till now.
 Well it certainly improves the types so i will include them